components/infobox/wikis/brawlstars/infobox_league_custom.lua

```diff
@@ -31,6 +30,8 @@ function CustomLeague.run(frame)
 	league.defineCustomPageVariables = CustomLeague.defineCustomPageVariables
 	league.addToLpdb = CustomLeague.addToLpdb
 	league.getWikiCategories = CustomLeague.getWikiCategories
```

Comment:
This should be removed right?

Reply:
yup, can be removed (so nil doesn't overwrite nil^^)